api_db.php

```diff
@@ -33,8 +47,14 @@ function SQLite3_connect($trytoreconnect)
 	}
 }
 
-$db = SQLite3_connect(true);
-
+if(strlen($DBFILE) > 0)
+{
+	$db = SQLite3_connect(true);
+}
+else
+{
+	die("No database available");
```

Comment:
Good checking if the database path exists, but won't it always be set because of the other if/else block?

Reply:
It is possible that `$FTLsettings["DBFILE"]` is set to an empty value (that is a legitime variant of telling `FTL` that you don't want the database) so this case can happen.